app/views/backend/product_nature_variants/show.html.haml

```diff
@@ -47,7 +47,7 @@
 
 
 = cobbles do |c|
-  - if resource.france_maaid? && (agent = Pesticide::Agent.find(resource.france_maaid))
+  - if resource.france_maaid? && (agent = Pesticide::Agent.find(resource.france_maaid.to_s))
```

Comment:
This is not a solution, the  `agent` returned by `Pesticide::Agent.find` will still be `Pesticide::Agent::Unknown` and will still raise the "Unknown method: `usages` for Unknown Struct" error.

The solution would rather be to add a :usages attribute on the structure and fill it with an empty array.